README.md

```diff
@@ -14,7 +14,7 @@ First download or clone this repository into directory `visualvm`. There are two
 
 ## Get the dependencies
   
-Then extract the [NetBeans 9.0 Dev platform and profiler binaries](visualvm/nb90_visualvm_11122017.zip) into directory `visualvm/visualvm` (should create `visualvm/visualvm/netbeans`).
+Then extract the [NetBeans 9.0 Dev platform and profiler binaries](visualvm/nb90_visualvm_01022018.zip) into directory `visualvm/visualvm` (should create `visualvm/visualvm/netbeans`).
```

Comment:
Replacing the ZIPs with new ones. The sizes seem comparable:
```
28303978 nb90_visualvm_01022018-ml.zip
27318977 nb90_visualvm_01022018.zip
27276414 nb90_visualvm_11122017-ml.zip
26286736 nb90_visualvm_11122017.zip
```
and doing `unzip nb90_visualvm_01022018.zip && ant run` seems to run the app fine.

Reply:
The size difference looks too big to me. Please investigate what caused this difference.